src/main/kotlin/no/nav/familie/ef/sak/iverksett/oppgaveterminbarn/ForberedOppgaverTerminbarnTask.kt

```diff
@@ -0,0 +1,47 @@
+package no.nav.familie.ef.sak.iverksett.oppgaveterminbarn
+
+import no.nav.familie.ef.sak.infrastruktur.featuretoggle.FeatureToggleService
+import no.nav.familie.prosessering.AsyncTaskStep
+import no.nav.familie.prosessering.TaskStepBeskrivelse
+import no.nav.familie.prosessering.domene.Task
+import no.nav.familie.prosessering.domene.TaskRepository
+import org.slf4j.LoggerFactory
+import org.springframework.stereotype.Service
+import java.time.LocalDate
+import java.time.format.DateTimeFormatter
+
+@Service
+@TaskStepBeskrivelse(
+        taskStepType = ForberedOppgaverTerminbarnTask.TYPE,
+        beskrivelse = "Oppretter oppgave for barn som fyller 1/2 eller 1 år",
+
+        )
```

Comment:
Burde kanskje få en annen beskrivelse
```suggestion
        beskrivelse = "Oppretter oppgave for ufødte terminbarn")
```